tensorflow/compiler/xla/service/gpu/gpu_conv_algorithm_picker.cc

```diff
@@ -254,7 +256,7 @@ StatusOr<AutotuneResult> GpuConvAlgorithmPicker::PickBestAlgorithm(
 
   StatusOr<AutotuneResult> result_or(InternalError("Unknown platform."));
   if (stream_exec_->platform_kind() == se::PlatformKind::kROCm) {
-    //result_or = PickBestAlgorithmNoCacheROCm(*instr, allocator, &stream);
+    result_or = PickBestAlgorithmNoCacheROCm(*instr, allocator, &stream);
```

Comment:
Also, per Google style naming convention, it should be `PickBestAlgorithmNoCacheRocm`, not `PickBestAlgorithmNoCacheROCm`.

Reply:
Sounds good.